### ISSUE TITLE
Testing input does not work as expected

### ISSUE THREAD

**Issue by untier** [vuejs/vue-test-utils#1049]
### Version
1.0.0-beta.11

### Reproduction link
[https://jsfiddle.net/a8untw0m/2/](https://jsfiddle.net/a8untw0m/2/)

### Steps to reproduce
Build a component, with an `input` in template, which has its `v-model` linked to a computed property and a prop `value` for usage as external `v-model`. In the setter of the input's model-property emit an input event, holding the inputted value. If the inputted value was an empty string, emit `null`.

Mount the component in a vue-test-utils wrapper, using a non-empty value-prop starting-value. Change the value of the `input`-element to "" and trigger the input event. Test the components value and the actual value of the `input`.

### What is expected?
The `value`-property of the component-vm should change to `null` and the value of the `input`-element should be an empty string. This is was happens if you use the component on an actual website (see example).

### What is actually happening?
The `value` prop of the component and the `value` of the `input` both remain at their last value.

**Comment by eddyerburgh** [comment]
You're using an old version of Vue Test Utils, since beta.12 the project has moved to `@vue/test-utils`.

@vue/test-utils now includes a `setValue` method which does what you expect. 

Please upgrade to that package, and check if your issue persists. If it does, please provide a minimal reproduction and I'll reopen the issue

**Comment by untier** [comment]
Thank you for your quick response.

I updated to 1.0.0-beta.26 and use `setValue`:

https://jsfiddle.net/a8untw0m/4/

This gets me the right `value` for the `input`-element (`""`). But the component-value and the value of the computed property are still not as I expect them to be.

EDIT:
As for the hint about a minimal example. In my opinion it is and I made it specfically for this isssue. Sorry if I don't meet your expectations here. I'm always open to suggestions.
